src/ipc/ipc-zephyr.c

```diff
@@ -188,6 +250,12 @@ int platform_ipc_init(struct ipc *ipc)
 
 	/* attach handlers */
 	intel_adsp_ipc_set_message_handler(INTEL_ADSP_IPC_HOST_DEV, message_handler, ipc);
+#ifdef CONFIG_PM
+	intel_adsp_ipc_set_suspend_handler(INTEL_ADSP_IPC_HOST_DEV,
+					   ipc_device_suspend_handler, ipc);
+	intel_adsp_ipc_set_resume_handler(INTEL_ADSP_IPC_HOST_DEV,
+					  ipc_device_resume_handler, ipc);
```

Comment:
I guess these are no-ops on platforms where RAM context is not maintained in D3? I.e. it's ok to still set them.